### ISSUE TITLE
Bug: stats `TimePass.parse` is missing when a processor returns no code blocks

### ISSUE THREAD

**Issue by ntnyq** [eslint/eslint#21293]
### Environment

Node version: v24.19.0
npm version: v11.17.0
Local ESLint version: v10.9.1
Global ESLint version: Not found
Operating System: darwin 24.6.0

### What parser are you using?

Default (Espree)

## Repro code

```js
import { ESLint } from "eslint";

const eslint = new ESLint({
    overrideConfigFile: true,
    stats: true,
    overrideConfig: [
        {
            files: ["**/*.txt"],
            processor: {
                preprocess() {
                    return [];
                },
                postprocess() {
                    return [];
                }
            }
        }
    ]
});

const [result] = await eslint.lintText("plain text", {
    filePath: "empty.txt"
});

console.log(JSON.stringify(result.stats, null, 2));
```

### What did you expect to happen?

Every entry in `stats.times.passes` should satisfy [ESLint's documented
`TimePass`](https://eslint.org/docs/latest/extend/stats) shape. 

```json
{
  "parse": {
    "total": 0
  },
  "fix": {
    "total": 0
  },
  "total": 0.293166
}
```

And in types file:

https://github.com/eslint/eslint/blob/87e0a082438264ad90b87fd74165ab4fd90f63ef/lib/types/index.d.ts#L1125-L1146

### What actually happened?

```json
{
  "times": {
    "passes": [
      {
        "fix": {
          "total": 0
        },
        "total": 0.286208
      }
    ]
  },
  "fixPasses": 0
}
```

### Link to Minimal Reproducible Example

https://github.com/ntnyq/repro-eslint-time-stats

### Participation

- [x] I am willing to submit a pull request for this issue.

### AI acknowledgment

- [x] I did not use AI to generate this issue report.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

**Comment by mdjermanovic** [comment]
Thanks for the report, I can reproduce this.

Since in this case there is no parsing, I think it really makes more sense to omit the property, as it currently works. We'd just update the docs and types to align with the current behavior.

@eslint/eslint-team thoughts?

**Comment by snitin315** [comment]
Agree with just updating docs & types. Marking this as accepted for the docs.

@ntnyq Feel free to send a PR updating docs & types to match current behaviour
